Nos/Extensions/String+Hex.swift

```diff
@@ -20,7 +22,7 @@ extension String {
                 let byte = Data([byteNum])
                 data.append(byte)
             } else {
-                fatalError("Could not decode byte to hex \(pair)")
+                return nil
```

Comment:
I changed this to `return nil` instead of fatalError because if you tried getting a hexDecoded value with a string with non-hex characters you'd end up with a fatal error, which didn't seem great. Also the isMultiple of 2 thing above is so you don't get an index out of bounds error when you try this with a string of odd length @mplorentz